### ISSUE TITLE
tools to "extract" answers from the ui ouput

### ISSUE THREAD

**Issue by Remi-Gau** [ReproNim/reproschema-py#8]
One thing that "should" part of this package would be a tool to read the content of the output from the ui and turn that into something the "average users" can actually "understand".

I am just afraid that a lot of people won't be satisfied with a "Look all your data is now a json-ld format matched to that specific schema. Isn't standardization great?" when the output is technically human readable but definitely not user friendly. :wink:

**Comment by satra** [comment]
i agree. @sanuann - can you add a function to reproschema-py to convert results to a csv?
